components/common/query/realm.md

```diff
@@ -0,0 +1,12 @@
+The request must include the following query parameters:
+
+| Field          | Type   | Description                        |
+| -------------- | ------ | ---------------------------------- |
+| Realm          | String | Defines the specific context or domain within which the Clearing-House request is being made. This header determines the environment or operational scope for the request. Allowed values include `ABS` (Automated Banking Systems), `CHM` (Clearing House Management), and `BCH` (Bank Clearing House). Each value represents a different area of functionality or service. |
```

Comment:
```suggestion
| Realm          | String | Defines the specific context or domain within which the Clearing-House request is being made. This header determines the environment or operational scope for the request. Allowed values include `ABS` (Access and Benefit-sharing Clearing House), `CHM` (Clearing House Mechanism), and `BCH` (Biosafety Clearing House). Each value represents a different area of functionality or service. |
```

Reply:
`suffix the realms with `-dev` to use the development and. `-trg` for training `